dockerfiles/README.md

```diff
@@ -72,6 +71,27 @@
   
   docker run -it onnxruntime-ngraph
   ```
+## ONNX Runtime Server (Preview)
+#### Linux 16.04
+
+1. Build the docker image from the directory that contains our server "onnxruntime_server"
```

Comment:
```suggestion
1. Build the docker image from the root directory of ONNXRuntime.
```

Reply:
As currently we copy the onnxruntime_server directly from the current folder into the docker. If we build the docker from the root directory, then we need to specify the relative path to the server which might changed due to our config is set to Debug/Release/ReleaseWithDebug info. For this reason I prefer to build it from the directory contains the server instead of the root folder.

Reply:
Ah my misunderstanding then